android/build.gradle

```diff
@@ -1,23 +1,22 @@
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
 
 buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+    // The Android Gradle plugin is only required when opening the android folder stand-alone.
+    // This avoids unnecessary downloads and potential conflicts when the library is included as a
+    // module dependency in an application project.
+    if (project == rootProject) {
+        repositories {
+            google()
+            jcenter()
+        }
+        dependencies {
+            classpath 'com.android.tools.build:gradle:3.4.1'
```

Comment:
@Krizzu - Quick question: I kept the version here the same as before, but could also update it to the latest stable version (`3.5.0`) at the same time. It would have no impact on any consumer app project. OK to make the change? Or do you prefer it to be in a separate PR?

Reply:
Yes, please upgrade to 3.5.0 in this PR

Reply:
Done.